### DISCUSSION TITLE
Problem in forcing data

### DISCUSSION THREAD

**Discussion by FuYingWang** [NCAR/hrldas#155] [Q&A] [answered]
I run the model as Note2_2D_NLDAS2.ipynb.
I download the raw data as '2. Download NLDAS2 Forcing'
![raw](https://github.com/NCAR/hrldas/assets/153582912/22b4b08b-42a2-4639-bc6c-c330399c9264)
And extract individial variables as 3.1
![ex](https://github.com/NCAR/hrldas/assets/153582912/d7d55053-f578-4a23-b0bd-8f7c5600665d)
![3](https://github.com/NCAR/hrldas/assets/153582912/d6f8d760-137b-4dbb-af03-c61dc54783c0)
Then, extract the initial-only fields as 3.2
![4](https://github.com/NCAR/hrldas/assets/153582912/a03399f4-8461-4c0e-8f59-ae887e5be215)
Also, did as 3.4.
I modify the namelist.input.NLDAS as 3.5
![namelist](https://github.com/NCAR/hrldas/assets/153582912/dc109533-3f87-4a41-9098-4531292641ab)
But, when I  enter    ./create_forcing.exe namelist.input.NLDAS, the problem like
![problem](https://github.com/NCAR/hrldas/assets/153582912/fd504b97-7e4d-49d0-a116-8a0479706d90)

**Comment by cenlinhe** [comment, marked as answer]
It looks like you did not download the NLDAS-2 model data for initial condition files (you are still using the NLDAS-2 forcing file). The model file is only needed for your initial time point to generate initial condition and it should be something like this file name: NLDAS_NOAH0125_H*. The forcing file has a name like NLDAS_FORA0125_H*

**Reply by cenlinhe** [reply]
@CharlesZheZhang It seems that you did not include the NLDAS-2 model data website link for downloading initial conditions in your tutorial note which may cause confusion. You only include and mention the NLDAS-2 forcing file downloading. Please update the note when you have time. Thank you!

**Reply by FuYingWang** [reply]
Thanks. According to your suggestion, I extract initial stats using 'NLDAS'_ NOAH0125_ After H * ', the problem was solved.

**Comment by CharlesZheZhang** [comment]
Hi @FuYingWang, please see this link for downloading the NLDAS_NOAH0125_H* for initial conditions (snow, soil moisture/temperature, canopy water): 
https://disc.gsfc.nasa.gov/datasets/NLDAS_NOAH0125_H_002/summary?keywords=NLDAS
You only need this file at the first time step of your simulation (2000-05-01_00). Then run `perl extract_nldas_init.perl` will create the initial data you will need for this step.

Many thanks,
Zhe

**Reply by FuYingWang** [reply]
Thanks. Based on your suggestion, I have successfully created the forcing data

**Comment by FuYingWang** [comment]
@cenlinhe Hello, there are some problem.
![V4}%OQ2Y171FGH`31DZM7{7](https://github.com/NCAR/hrldas/assets/153582912/66cd58ae-8e47-4263-a6a8-402f6ce12623)

This are my namelist.hrldas
![9TM)$BM%@ZN14950OIKO`JT](https://github.com/NCAR/hrldas/assets/153582912/f29555ec-d64c-4035-acc6-afc5196482a5)

**Reply by cenlinhe** [reply]
what is your error message? Did the model crash at some point? I only see your snapshot of parameter table and namelist but not sure what is your problem.

**Reply by FuYingWang** [reply]
![2XZWMU0J(9$XG$SLTJTDVIS](https://github.com/NCAR/hrldas/assets/153582912/f7d5517c-06db-473a-8f02-2f5cc1d03d32)

**Reply by CharlesZheZhang** [reply]
While the error message is pointing to the NoahmpTable.TBL, line 2, position 0, but it is not showing anything weird in the table as you provided. I haven't seen this type of error before. 
This could be related to your fortran compiler. What's your fortran compiler? You can try another compiler, we now support three (intel, pgi, and gfortran), see if it can help you with this case.
